File: java/swagger-demo/src/main/java/com/opusmagus/OrderController.java
package com.opusmagus;

import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
public class OrderController {
    @Autowired ProcessOrderCommand processOrderCommand;

    @RequestMapping("/order/process")
    public Object ProcessOrder() throws Exception {
        return processOrderCommand.Execute(new ProcessOrderInputModel());
    }
}

File: dotnet/web-app/Controllers/OrderController.cs
using Microsoft.AspNetCore.Mvc;

namespace com.opusmagus
{
    public class OrderController : Controller
    {
        private ProcessOrderCommand processOrderCommand;

        public OrderController(ProcessOrderCommand processOrderCommand) {
            this.processOrderCommand = processOrderCommand;
        }

        [HttpGetAttribute("/order/process")]        
        public dynamic ProcessOrder(){
            return processOrderCommand.Execute(new ProcessOrderInputModel());
        }
    }
}